mongo/mongo.go

```diff
@@ -94,17 +114,9 @@ func (m *Mongo) GetCollections(ctx context.Context, queryParams collections.Quer
 	var values []models.Collection
 
 	if queryParams.Limit > 0 {
-		iter := q.Skip(queryParams.Offset).Limit(queryParams.Limit).Iter()
-
-		defer func() {
-			err := iter.Close()
-			if err != nil {
-				log.Error(ctx, "error closing iterator", err)
-			}
-		}()
-
-		if err := iter.All(&values); err != nil {
-			if err == mgo.ErrNotFound {
+		err = q.Skip(queryParams.Offset).Limit(queryParams.Limit).IterAll(ctx, &values)
+		if err != nil {
+			if dpMongoDriver.IsErrNoDocumentFound(err) {
```

Comment:
you will not receive this error you will get an empty array

Reply:
Removed check for `ErrNoDocumentFound`